hack/make/.ensure-frozen-images

```diff
@@ -4,7 +4,7 @@ set -e
 # this list should match roughly what's in the Dockerfile (minus the explicit image IDs, of course)
 images=(
 	busybox:latest
-	hello-world:latest
+	hello-world:frozen
```

Comment:
All else failing meaning that we don't have the pre-downloaded images from the `Dockerfile` (which is the ideal case) _and_ we don't have `curl` in order to use the download script directly (which is the second-best option).  Our last-resort fallback in this file is to `docker pull` (see line 39 of this file).

I'd be OK making that a hard fail instead, but I think that would warrant a little bit more discussion, because `curl` would essentially become a build-time dep for anyone wanting to run the integration-cli tests.